projects/assets/README.md

```diff
@@ -0,0 +1,24 @@
+# Assets
```

Comment:
We should override default README with a description of this particular library.